chapter_post_and_database.asciidoc

```diff
@@ -1889,4 +1908,7 @@ The scratchpad to-do list::
 place to write down things that occur to us as we're coding, so that
     we can finish up what we're doing and come back to them later.
 
+// SEBASTIAN: (idea) alternative to maintaining a scratchpad could be to write empty unit tests without implementation.
+//    Such "tests prototypes" could be skipped initially until we work on them.
```

Comment:
i've always like this idea in theory, but i've never found myself actually bothering to do it in practice, or not more than once or twice -- have you ever got into the habit?